src/actions/voting.js

```diff
@@ -90,7 +90,7 @@ const handleVoteError = ({ error }) => {
  * cleans the pending state
  */
 export const votePlaced = ({
-  passphrase, account, votes, secondPassphrase, goToNextStep,
+  passphrase, account, votes, secondPassphrase, callback,
```

Comment:
I know this was already like this, but could be a way to split this function?, right now the code looks good but it has so many lines of code now, that start to looks bad

Reply:
I agree this needs a clean-up and it will be done in https://github.com/LiskHQ/lisk-hub/pull/2099